src/ispypsa/translator/temporal_filters.py

```diff
@@ -0,0 +1,182 @@
+from datetime import datetime, timedelta
+
+import pandas as pd
+
+from ispypsa.config.validators import ModelConfig
+
+
+def apply_snapshot_filter(time_series_data, snapshot):
```

Comment:
Arguments missing type.

 I think the use of `snapshot` in this module and in `snapshot.py` is a bit confusing since snapshots in ISPyPSA can refer to a complete time series (as produced by snapshot.py) or what is attached to a network in PyPSA (which is filtered as per buses.py and generators.py).

Few options:
1. As per the refactor PR, rename the func in snapshot.py to reflect that it is create a full range snapshot or time series
2. The, in this file, you could keep the function name as is. Or, change the function name to `time_series_filter` and change `snapshot` to `snapshot_filter` or `retained_datetimes` or something like that

Reply:
Good suggestion. Made this change.